tests/Unit/Evolution/DiscontinuousGalerkin/Actions/Test_NormalCovectorAndMagnitude.cpp

```diff
@@ -203,13 +206,15 @@ void check_normal_covector_quantities(
             *box));
 
     if constexpr (UseFlatSpace) {
-      CHECK_ITERABLE_APPROX(
-          (DataVector{fields_on_face.number_of_grid_points(), 1.0}),
-          get(magnitude(
-              get<evolution::dg::Tags::InternalFace::NormalCovector<Dim>>(
-                  *get<evolution::dg::Tags::InternalFace::
-                           NormalCovectorAndMagnitude<Dim>>(*box)
-                       .at(direction)))));
+      const auto& normal_vector =
```

Comment:
Should this be `normal_covector`? The tag on the RHS is covector. I guess in flat space it makes little difference, but might still be more consistent to go with covector? Or perhaps I've missed something.